### ISSUE TITLE
There is no artifact to deploy in staging directory when deploy.

### ISSUE THREAD

**Issue by EricZhangzf** [microsoft/azure-maven-plugins#967]
### Plugin name and version
azure-webapp-maven-plugin 1.9.0-SNAPSHOT
### Plugin configuration in your `pom.xml`
```
<build> 
    <plugins> 
      <plugin>
        <groupId>com.microsoft.azure</groupId>
        <artifactId>azure-webapp-maven-plugin</artifactId>
        <version>1.9.0-SNAPSHOT</version>
        <configuration>
          <schemaVersion>V2</schemaVersion>
          <resourceGroup>your-resource-group</resourceGroup>
          <appName>your-appname</appName>
          <pricingTier>P1v2</pricingTier>
          <region>eastus</region>
          <runtime>
            <os>linux</os>
            <javaVersion>jre8</javaVersion>
            <webContainer>tomcat 8.5</webContainer>
          </runtime>
          <deployment>
            <resources>
              <resource>
                <directory>${project.basedir}/target</directory>
                <includes>
                  <include>*.war</include>
                </includes>
              </resource>
            </resources>
          </deployment>
        </configuration>
      </plugin>
    </plugins> 
  </build> 
```
### Actual behavior
![image](https://user-images.githubusercontent.com/44559777/71461848-49ac1180-27ec-11ea-99bb-ba7e137c92d2.png)

### Steps to reproduce the problem
1.  mvn azure-webapp:config to config your pom.xml like above.
2.  mvn clean package
3.  mvn azure-webapp:deploy

**Comment by EricZhangzf** [comment]
Test pass on 1.9.0

**Comment by EricZhangzf** [comment]
Still repro on 1.10.0-SNAPSHOT three OS, So reopen it.

**Comment by Flanker32** [comment]
Fixed in #1004
